src/wesnoth.cpp

```diff
@@ -941,7 +941,7 @@ static void restart_process(const std::vector<std::string>& commandline)
 #define error_exit(res) return res
 #endif
 
-#ifdef __APPLE__
+#ifdef __IPHONEOS__
```

Comment:
The question on XCode project is still unclear to me.
XCode project itself and iOS-specific files like icons is not a big deal.
What is a question, that is dependencies. Do we need a copy of 
`bzip2-1.0.1
cairo-1.10.0
flac-1.2.1
fontconfig-2.8.0
fonteditor
freetype-2.4.12
gettext-0.18
gif
include
iOS_boost_1_60
jpeg-8b
lib
libogg-1.3.1
libpng-1.4.4
libvorbis-1.3.3
lua
mikmod
pango-1.28.3
pixman-0.21.2
SDL2_image-2.0.1
SDL2_mixer-2.0.1
SDL2_ttf-2.0.14
SDL2-2.0.5
tremor
xml2` in the mainline git? 
Having tried, I have hard times reducing it to less than 500M.

Boost binary alone takes 56M archived, because I prefer to check in a binary, as it's a nontrivial task to build it for iOS, and build script breaks with new SDK versions. Plus 100M headers... probably `phoenix fusion spirit` can be excluded, but is worth the hassle?..
Then, `glib` with its dependencies. 150M binaries, though, a tiny build script. But again, I prefer to keep binaries.

So, is it OK to leave an iOS project file in a separate project?